src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        Color[] colors = Color.values();
+        if (colors.length == 0) {
+            throw new IllegalStateException("No colors defined");
+        }
```

Comment:
Cannot say it's mistake, it's definitely not, but I have never seen intentionally empty enums so this check seems excessive